openapi.yaml

```diff
@@ -330,15 +330,15 @@ components:
         quotaMaxBytes:
           type: integer
           format: int64
-          description: "Maximum storage space used by the node"
+          description: "Maximum storage space (in bytes) used by the node on the hard drive."
         quotaUsedBytes:
           type: integer
           format: int64
-          description: "Amount of storage space currently in use"
+          description: "Amount of storage space (in bytes) currently in use by availabilities for existing files."
```

Comment:
```suggestion
          description: "Amount of storage space (in bytes) currently used for storing files in Codex's local repository."
```

The `usedBytes` can be utilized not only by "availabilities", but when you simply fetch content from network (with the `GET /data/{cid}/network` call), the content is stored locally in the repository for some time before the data expires and is cleaned up automatically by the "garbace-collection" (GC) process. Hence, let’s remove the "by availabilities" parts.